time stamp

# base (shared) values
int32 test_number
bool default_control
string flight_mode
bool test_finished
bool coord_ok

# StatusPrimary (customize as needed!)
string control_mode  # example: a control_mode specified by your tests
bool regulate_finished  # example: a variable controlling regulation of the robots
string description  # example: descriptive information for telemetry
